test/get_previous_releases.py

```diff
@@ -96,8 +105,11 @@ def download_binary(tag, args) -> int:
     if match:
         bin_path = 'bin/bitcoin-core-{}/test.{}'.format(
             match.group(1), match.group(2))
+    platform = args.platform
+    if float(tag[1:]) < 23 and (platform == "x86_64-apple-darwin" or platform == "aarch64-apple-darwin"):
```

Comment:
The comparison `float(tag[1:]) < 23`  looks ugly ... doesn't it break when tag is e.g. "v0.21.0"?

I think the simple `tag < "v23"` or `tag[1:] < "23"` does the trick, no?

Also this is Python so we can use ` platform in ...`

```suggestion
    if tag < "v23" and platform in ["x86_64-apple-darwin", "aarch64-apple-darwin"]:
```

Reply:
Done!